### ISSUE TITLE
Interface to support multiple sponsors on a document

### ISSUE THREAD

**Issue by sethetter** [opengovfoundation/madison#891]
Most of the code is written to support multiple sponsors on a document as this is the intended route to go eventually. The issue now is to figure out how the interface should be implemented.

First thought I have is to have a list of checkboxes. It will show all sponsors that the current user has access to (groups they are a part of, or themselves as individual sponsor). If the document has a sponsor that the user *does not* have access to, the checkbox will be checked and disabled.

Thoughts?

**Comment by krues8dr** [comment]
Even with the work on #888 to make everything groups-only, this is still a rather complicated interface.  Users need to be able to add new users and groups to the document, and the Users part in particular seems complicated overall.  If they add a user and they're not an independent sponsor user, then what?  I assume that user gets an invite, then we have to take them through the sponsor verification process and then back to the document?  Or we just let users add them to an existing group on the same page somehow, probably reusing the add user to group functionality we have elsewhere?  There are a lot of subtle interactions here and I'm not 100% sure of how they all should work together.  I recommend punting on this for now.

**Comment by sethetter** [comment]
@krues8dr You're right, there's a lot of nuances that could be included. I think a simple first step would be to only show sponsors that the current user is eligible to add and restrict them from removing sponsors that are on the document that the user does not have access to.

The ability to add users to the document that aren't individual sponsors yet would be nice, but I think in the meantime we can just only accept adding sponsors to the document that would otherwise be currently eligible.
